src/__tests__/store/message/reducer.test.ts

```diff
@@ -0,0 +1,176 @@
+import { messageActionTypes, IMessageState } from "../../../store/message/types";
+import { messageReducer } from "../../../store/message/reducer";
+
+const initialState: IMessageState = {
```

Comment:
@xhyzzZ there are currently no reducer modifications on the message state.  you can create a test bed by ensuring there are no mods to the initial state. In this case all tests can be similar to the initial state test. you don't need any testState in this file since there are no modification of the initial state.